lib/reline/unicode.rb

```diff
@@ -160,24 +160,29 @@ def self.split_by_width(str, max_width, encoding = str.encoding)
     width = 0
     rest = str.encode(Encoding::UTF_8)
     in_zero_width = false
+    seq = String.new(encoding: encoding)
```

Comment:
What does `seq` mean in this context?

Reply:
escape sequences
It's used to accumulate escape sequences of text color, background color, text style etc.